templates/vault.systemd.erb

```diff
@@ -23,7 +23,11 @@ CapabilityBoundingSet=CAP_SYSLOG
 NoNewPrivileges=yes
 <% else -%>
 SecureBits=keep-caps
+<% if scope['::systemd_version'] > 229 -%>
```

Comment:
There's something up here looking at the test failures on Travis.
I'm not sure if the `confine` is working as expected.

```
 error during compilation: Evaluation Error: Error while evaluating a Function Call, Failed to parse template vault/vault.systemd.erb:
 Filepath: /home/travis/build/jsok/puppet-vault/spec/fixtures/modules/vault/templates/vault.systemd.erb
 Line: 26
 Detail: undefined method `>' for nil:NilClass
```
https://travis-ci.org/jsok/puppet-vault/jobs/290455939

Reply:
Yeah I am still on it. Just using Travis to test it.